### PR TITLE
[@types/jquery] .appendTo() should accept a document fragment as an argument

### PR DESCRIPTION
**Pull request by terrymun**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://api.jquery.com/appendto/
    Further elabouration: since `.appendTo()` accepts a jQuery object, we should not restrict to the method accepting only `Element`, but also any kind of `Node`.
- [x] ~Increase the version number in the header if appropriate.~ jQuery minor version remains the same.
- [x] ~If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.~ File already exists.

### CODE REVIEW

**Comment by terrymun** [comment]
The TravisCI build is failing but the error seems to be coming from other typings instead of jQuery's. Do we have a course of action to resolve this issue?

File: types/jquery/index.d.ts [hunk 1]
```diff
@@ -3487,7 +3488,7 @@ interface JQuery<TElement = HTMLElement> extends Iterable<TElement> {
      * @see \`{@link https://api.jquery.com/appendTo/ }\`
      * @since 1.0
      */
-    appendTo(target: JQuery.Selector | JQuery.htmlString | JQuery.TypeOrArray<Element> | JQuery): this;
+    appendTo(target: JQuery.Selector | JQuery.htmlString | JQuery.TypeOrArray<Node> | JQuery): this;
```
Comment (leonard-thieu) [inline types/jquery/index.d.ts]: From my [testing](https://jsfiddle.net/leonard_thieu/xpvt214o/774608/), it looks like it doesn't apply to all `Node` types. Maybe it's more appropriate to make it `Element | DocumentFragment`?

**Review by leonard-thieu — CHANGES_REQUESTED** [review]
This change should be applied to `prependTo` too.

https://jsfiddle.net/leonard_thieu/xpvt214o/774683/

File: types/jquery/index.d.ts (hunk 1 above)
Comment (terrymun) [reply types/jquery/index.d.ts]: @leonard-thieu True that. In that case, we should make sure `appendTo()` and `prependTo()` accepts `Element | DocumentFragment`. Apologies for the slow reply, had a really busy week :) I'll submit the revised PR today.

**Review by leonard-thieu — APPROVED** [review]

**Comment by leonard-thieu** [comment]
Test failure fixed by #28947. Please pull in the latest changes.

**Comment by terrymun** [comment]
@leonard-thieu Thanks for the heads up. I wonder what's causing so many tests to fail recently: hope the latest upstream merge will fix it for good. I've just pushed.

**Pull request merged**